python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -287,4 +287,4 @@ def upper(x):
         )
         resolvers.append(ResolverSpec(None, col_type, [MetricSpec(UdfMetric, config)]))
 
-    return resolvers
+    return DeclarativeSchema(non_udf_resolvers + resolvers)
```

Comment:
Consider splitting into separate functions or passing the other `DeclarativeSchema` ctor arguments.

Reply:
we can do more refactors in follow up, this should cover initial cases, thanks!